zcash_client_backend/Cargo.toml

```diff
@@ -13,11 +13,13 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1.52"
```

Comment:
I see you were experimenting with `async-trait` version, What was the problem?

Reply:
yes, I needed to use a version that's compatible with `mm2` as the version is locked